server/game/cards/locations/07/blackbetha.js

```diff
@@ -0,0 +1,57 @@
+const DrawCard = require('../../../drawcard.js');
+
+class BlackBetha extends DrawCard {
+    //Just used Iron Fleet Scout as template.  Overly verbose, but easy to understand.  Could be refactored into something more concise (a la The Valyrian).
+
+    setupCardAbilities(ability) {
+        this.persistentEffect({
+            match: card => card.name === 'Ser Davos Seaworth',
+            effect: ability.effects.addKeyword('renown')
+        });
+        this.action({
+            title: 'Kneel to give attacking character +X STR',
+            cost: ability.costs.kneelSelf(),
+            condition: () => this.game.currentChallenge && this.game.currentChallenge.attackers.length >= 1,
+            method: 'kneel'
+        });
+    }
+
+    kneel(player) {
+        this.game.promptForSelect(player, {
```

Comment:
A bunch of older cards use these manual prompts still, but we've since switched to using a target property inside the action to handle cases like these, which would look something like this:

```javascript
            target: {
                activePromptTitle: 'Select character to gain STR',
                cardCondition: card => this.returnCondition(card)
            },
```
(See the card implementation docs for more information.)
This prevents the ability from initiating if there's no legal target, meaning you can get rid of the current condition check, and instead check if the defending player has at least 1 kneeling character (so the effect will actually change the game state, another requirement for it to be able to initiate).